include/mbgl/util/projection.hpp

```diff
@@ -13,7 +13,7 @@ namespace mbgl {
 class Projection {
 public:
     // Map pixel width at given scale.
-    static double worldSize(double scale) {
+    static constexpr double worldSize(double scale) {
```

Comment:
Nothing special besides the fact `util::tileSize` is already known in compile time - as requested below I'll subtract this cosmetic change from this PR 👍